app/models/miq_cockpit_ws_worker.rb

```diff
@@ -14,8 +14,7 @@ def self.can_start_cockpit_ws?
   end
 
   def self.should_start_worker?
-    return false unless has_required_role?
-    can_start_cockpit_ws?
+    super && can_start_cockpit_ws?
   end
 
   def self.sync_workers
```

Comment:
can we delete this method now that super listens to `should_start_worker?`

I hesitate only because `@workers = ...` looks like a hack here. But I'm not totally sure why this fixes things